src/picknik_ur_site_config/README.md

```diff
@@ -1,5 +1,7 @@
 # picknik_ur_site_config
 
-A MoveIt Studio site configuration for UR5e based on the picknik_ur_base_config package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config). This package follows the required structure described in [Creating a New Site Config Package](https://docs.picknik.ai/en/stable/concepts/config_package/config_package.html#creating-a-new-site-config-package), and may be used to add environments, behaviors, and objectives on top of the UR5e base configuration specified in `site_config.yaml`
+A MoveIt Studio site configuration for UR5e based on the `picknik_ur_base_config` package available [here](https://github.com/PickNikRobotics/ur5e_sim_base_config).
+
+This package follows the required structure described in [Creating a New Site Config Package](https://docs.picknik.ai/en/stable/concepts/config_package/config_package.html#creating-a-new-site-config-package), and may be used to add environments, behaviors, and objectives on top of the UR5e base configuration specified in `site_config.yaml`
```

Comment:
I think everywhere else in the tutorials Objectives and Behaviors are capitalized